Change tracking: Move setup higher on the page. (#11947)
newrelic/docs-website: src/content/docs/change-tracking/change-tracking-introduction.mdx

Our change analysis UIs provide details and insights about each of the changes you track, such as faceted errors, log attribute trends, related issues and anomalies, and calculated impacts on key signals. With these insights, you'll be able to understand the impact of changes on system performance and quality. Change tracking metadata, such as commit SHAs, changelogs, and deep links, will help you resolve issues more quickly.

> To get acquainted with various aspects of change tracking, watch the video below or see our [feature highlights](#feature-highlights). If you're ready to capture and analyze changes across your stack, go right to [Start tracking changes](#start-tracking).

Reviewer: ```suggestion
To get acquainted with various aspects of change tracking, watch the video below or see our [feature highlights](#feature-highlights). If you're ready to capture and analyze changes across your stack, see [Start tracking changes](#start-tracking).
```

<Video id="ikRUKJf7hyU" type="youtube" />
<figcaption>
  Here's a six-minute video explaining change tracking.
</figcaption>

## Start tracking your changes [#start-tracking]

To begin capturing and analyzing changes, make sure you have set up a free New Relic account.
<InlineSignup/>

Once you have an account, choose one of these options to get started.

> * [NerdGraph (GraphQL) API](/docs/change-tracking/change-tracking-graphql): You can use GraphQL mutations to designate what changes to track and then run queries to list details about what you're tracking.

Reviewer: Do we need to link to a resource that explains "GraphQL mutations"? I have no clue what this concept means.

* [New Relic CLI](/docs/change-tracking/change-tracking-cli): Use our command line interface to designate what changes you want to track.

> * [Jenkins](/docs/change-tracking/ci-cd/change-tracking-jenkins): Our plugin makes it easy for you to use Jenkins for tracking changes.

Reviewer: ```suggestion
* [Jenkins](/docs/change-tracking/ci-cd/change-tracking-jenkins): Our plugin simplifies using Jenkins for tracking changes.
```

* [GitHub Actions](/docs/change-tracking/ci-cd/change-tracking-github-actions): We have tips about how to configure GitHub actions in your CI/CD pipeline.

> After you capture changes, start using them to contextualize your telemetry, performance, and quality data. Learn more about this in [How to view and analyze your changes in New Relic](/docs/change-tracking/change-tracking-view-analyze/).

Reviewer: ```suggestion
After you capture changes, use them to contextualize your telemetry, performance, and quality data. Learn more about this in [How to view and analyze your changes in New Relic](/docs/change-tracking/change-tracking-view-analyze/).
```
